### PR TITLE
Cybersecurity New Topic: Red team

### PR DESCRIPTION
**Pull request by jonchery**
### Description

Created new cybersecurity folder and content, Red team.

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/cybersecurity/concepts/red-team/red-team.md
Comment (SSwiniarski) [inline content/cybersecurity/concepts/red-team/red-team.md:7]:
```suggestion
  - 'Cybersecurity'
  - 'Ethical Hacking'
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@jonchery I did a first review and left a few comments.

**Review by SSwiniarski — APPROVED** [review]
@jonchery looks good for a second review.

File: content/cybersecurity/concepts/red-team/red-team.md
Comment (HishamT) [inline content/cybersecurity/concepts/red-team/red-team.md:17]:
```suggestion
A Red Team might seek to carry out a penetration test to model real-world attack techniques and methods. It can usually gain access through the theft of user credentials and social engineering techniques. Red teams are usually composed of highly experienced security professionals or ethical hackers. Red teams can help organizations identify misconfigurations, strengthen network security, and increase awareness among employees as to the risk of human error which may compromise the organization’s security.
```

**Review by HishamT — COMMENTED** [review]
Hey @jonchery, I have finished the second review and left a comment.

**Comment by SSwiniarski** [comment]
@jonchery, another edit that needs to be made: All the headers need to have line breaks after, and all the headers need trailing punctuation (the `:` colon) removed. That should help this pass the checks.

**Comment by SSwiniarski** [comment]
@jonchery In the interests of getting this merged I applied the above edits. Once @HishamT reviews and approves, we can merge.

**Review by HishamT — APPROVED** [review]
LGTM!

**Comment by yangc95** [comment]
👏 @jonchery live link: https://www.codecademy.com/resources/docs/cybersecurity/red-team

Screenshot 2023-03-19 at 3 13 34 PM

> It looks as if you're anonymous. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.
> 
> To add your GitHub username to your Codecademy profile:
> 
> 1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:
> 
> image
> 
> 2.) Then log out and log back into Codecademy using your GitHub login:
> 
> githublogin
>
> If the steps above do not work, it may be due to an email or local configuration issue.
>
> In addition, we may reach out in the future if any adjustments to these entries are required.

cc: @HishamT @SSwiniarski

**Pull request merged**
